controllers/lvmcluster_controller_integration_test.go

```diff
@@ -189,7 +190,15 @@ var _ = Describe("LVMCluster controller", func() {
 
 			// delete lvmVolumeGroupNodeStatus as it should be deleted by vgmanager
 			// and if it is present lvmcluster reconciler takes it as vg is present on node
-			Expect(k8sClient.Delete(ctx, lvmVolumeGroupNodeStatusIn)).Should(Succeed())
+
+			// we will now remove the node which will cause the LVM cluster status to also lose that vg
+			Expect(k8sClient.Delete(ctx, nodeIn)).Should(Succeed())
+			// deletion of LVMCluster CR
+			Eventually(func() bool {
```

Comment:
Is this just testing that the controller doesn't hang if the node is removed?

Reply:
Basically this tests that the LVMVolumeGroupNodeStatus is removed "Eventually" after the node gets removed from the cluster. This is basically the test of the control loop that was introduced. Previously the test directly removed the NodeStatus object